### PR TITLE
android: add an option to ignore volume keys or use them to turn pages

### LINKED ISSUE #1760: Bluetooth does not work.
Hi

Bt controls do not work when Koreader is open. basically i can hear the music through bt but the play butttons do not work when koreader is in focus. When i get back to android desktop, bt controls start working again.

I am using lollipop 5.1

### LINKED ISSUE #3374: FR: Use Android bluetooth volume control for page turn
Would love to be able to use a bluetooth volume control remote for page turns with Android koreader.

### PR DESCRIPTION
**Pull request by pazos**
Related to #5136 

Requires https://github.com/koreader/koreader-base/pull/938
Requires https://github.com/koreader/android-luajit-launcher/pull/162

Fixes #1760 
Fixes #3374

### CODE REVIEW

File: frontend/ui/elements/common_settings_menu_table.lua [hunk 1]
```diff
@@ -175,6 +175,17 @@ if Device:isAndroid() then
     -- screen timeout options, disabled if device needs wakelocks.
     common_settings.screen_timeout = require("ui/elements/screen_android"):getTimeoutMenuTable()
 
+    -- volume key events
+    common_settings.android_volume_keys = {
+        text = _("Use volume keys to turn pages"),
```
Comment (Frenzie) [inline frontend/ui/elements/common_settings_menu_table.lua]: `(Use) volume key page turning`?

**Comment by pazos** [comment]
@Frenzie: changed the name to `"Volume key page turning"`. Please modify as you wish and merge it when ready!

**Comment by Frenzie** [comment]
That's good by me. You updated this PR with the recently merged base/luajit-launcher stuff?

**Comment by pazos** [comment]
>  You updated this PR with the recently merged base/luajit-launcher stuff?

Yes!

**Pull request merged**